package.json

```diff
@@ -9,14 +9,10 @@
     "node": ">=10.15.0"
   },
   "scripts": {
-    "ctest": "cross-env BROWSER=chromium node --unhandled-rejections=strict test/test.js",
-    "ftest": "cross-env BROWSER=firefox node --unhandled-rejections=strict test/test.js",
-    "wtest": "cross-env BROWSER=webkit node --unhandled-rejections=strict test/test.js",
-    "test": "cross-env node --unhandled-rejections=strict test/test.js",
-    "ccoverage": "cross-env COVERAGE=true BROWSER=chromium node --unhandled-rejections=strict test/test.js",
-    "fcoverage": "cross-env COVERAGE=true BROWSER=firefox node --unhandled-rejections=strict test/test.js",
-    "wcoverage": "cross-env COVERAGE=true BROWSER=webkit node --unhandled-rejections=strict test/test.js",
-    "coverage": "cross-env COVERAGE=true node --unhandled-rejections=strict test/test.js",
+    "ctest": "cross-env BROWSER=chromium npm run jest",
+    "ftest": "cross-env BROWSER=firefox npm run jest",
+    "wtest": "cross-env BROWSER=webkit npm run jest",
+    "test": "npm run ctest && npm run ftest && npm run wtest",
```

Comment:
We'll have to change travis, appveyor and circle to use ctest/ftest/wtest separately, otherwise they will always stop after ctest failing.

Reply:
Let's follow up!